docs/clients/s3.md

```diff
@@ -67,3 +67,14 @@ This information is required by AWS, and cannot be guessed by AsyncAws.
 If `ContentLength` is absent, AsyncAws will read the output before sending the
 request which could have a performance impact.
 
+## Virtual Hosted-Style Requests
+
+When calling AWS endpoints, Async Aws uses [Virtual Hosted-Style Requests](https://docs.aws.amazon.com/AmazonS3/latest/dev/VirtualHosting.html):
+The bucket name is part of the endpoint's host. To disable this behaviors, use
+the parameter `s3PathStyleEndpoint` when initializing the client.
+
+```php
+use AsyncAws\S3\S3Client;
+
+$s3 = new S3Client(['pathStyleEndpoint' => false]);
```

Comment:
Is it now "s3PathStyleEndpoint" or "pathStyleEndpoint"? 🤔

Reply:
good catch!!! thanks